app/typer/sanity/sanity.ts

```diff
@@ -0,0 +1,51 @@
+import { PortableTextBlock } from '@portabletext/types';
+
+export interface SanityDokument {
+  _createdAt: string;
+  _rev: string;
+  _type: string;
+  _id: string;
+  api_navn: ESanityApiKey;
+  visningsnavn: string;
+  /*  nb: CustomSanityTyper.CUSTUM_BLOCK;
+  nn: CustomSanityTyper.CUSTUM_BLOCK;
+  en: CustomSanityTyper.CUSTUM_BLOCK; */
```

Comment:
Det kan være litt skummelt å legge inn kommentarer på framtidig kode. Det veldig lett å glemme å fikse det senere og kan være vanskelig for nestemann å tolke

Reply:
Jeg tror dere kan bruke typen `PortableTextBlock` her
```
nb:  PortableTextBlock;
nn:  PortableTextBlock;
en:  PortableTextBlock;
```

Reply:
Okay og det er endret nå [2343bbb](https://github.com/bekk/nav-familie-endringsmelding/pull/13/commits/2343bbb3576b2b40055fcf696c663f31d3e6c282)